docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -522,13 +522,35 @@
    "outputs": [],
    "source": [
     "pos = sc.zeros(sizes=binned_logical.sizes, dtype=sc.DType.vector3, unit='mm')\n",
-    "pos.fields.x = binned_logical.bins.coords['voxel_x'].bins.mean()\n",
-    "pos.fields.y = binned_logical.bins.coords['voxel_y'].bins.mean()\n",
-    "pos.fields.z = binned_logical.bins.coords['voxel_z'].bins.mean()\n",
+    "# We 'pop' the coordinates. If desired they could be kept by using\n",
+    "# __getitem__ instead\n",
+    "pos.fields.x = binned_logical.bins.coords.pop('voxel_x').bins.mean()\n",
+    "pos.fields.y = binned_logical.bins.coords.pop('voxel_y').bins.mean()\n",
+    "pos.fields.z = binned_logical.bins.coords.pop('voxel_z').bins.mean()\n",
     "binned_logical.coords['position'] = pos\n",
     "binned_logical"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d7f4ee17-09a3-45a6-a86b-a7579f587a10",
+   "metadata": {},
+   "source": [
+    "We have removed three event-based coordinates (`voxel_x`, `voxel_y`, and `voxel_z`) and replaced it by a single bin-base coordinate, `position`.\n",
```

Comment:
'bin-base coordinate'? I don't think we call it that anywhere else. Wouldn't it just be 'bin-coordinate'?

Reply:
```suggestion
    "We have removed three event-based coordinates (`voxel_x`, `voxel_y`, and `voxel_z`) and replaced it by a single bin-coordinate, `position`.\n",
```